stencil-library/src/components/refund-form/refund-form.scss

```diff
@@ -0,0 +1,99 @@
+@import 'bootstrap-grid';
+@import 'bootstrap-utilities';
+
+:host {
+  @include host-base-styles;
+  display: block;
+}
+
+.info-text-container {
```

Comment:
I think all of the custom styles in here can be achieved using bootstrap grid, utilitly classes, and the [border classes](https://getbootstrap.com/docs/5.3/utilities/borders/#border)

Reply:
Perfect, fixed!